### DISCUSSION TITLE
I made a color guessing game using color-names

### DISCUSSION THREAD

**Discussion by kiru** [meodai/color-names#168] [Show and tell]
![color-guesser](https://github.com/meodai/color-names/assets/2451608/37bda891-9dd0-4c75-82e0-6ae24d04ba8f)

Hi there 👋, 

I created a simple game where you have to guess the colors based on the name: [ColorGuesser.com](https://colorguesser.com/)

So far, I have a working prototype. Many things could be improved or added (I have written about this [here](https://kiruio.substack.com/i/126414175/incubating-colorguessercom)).

I am open for any feedback. 

Thank you very much, 
Kiru

**Comment by meodai** [comment]
Hey! I love it. Turns out, that I am not too bad at it ;)

Anyway, I do have some feedback:

1. I would add a difficulty level. That might increase of the duration of the game? By starting with simple color name lists and end the game with the full name list: https://github.com/meodai/color-name-lists 
2. Instead of having the list in your code, you could just use the public API: https://github.com/meodai/color-name-api (it comes with all the different lists)
3. After the user has guessed I would visually show how far or close the guess was (similar to geo guesser)

**Reply by kiru** [reply]
Hi David, 
Thank you very much for testing and the suggestions. Here are my comments: 

1/2)That's a great idea! I didn't know there was an API, definitely look into it!
3) Added to my list! 

I got some great suggestion so far (also from other sides), will add the easiest one and then see for the rest.

Cheers, 
Kiru

**Comment by The-Ghost-Crew** [comment]
> ![color-guesser](https://private-user-images.githubusercontent.com/2451608/251094381-37bda891-9dd0-4c75-82e0-6ae24d04ba8f.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjgxMjUxNDgsIm5iZiI6MTc2ODEyNDg0OCwicGF0aCI6Ii8yNDUxNjA4LzI1MTA5NDM4MS0zN2JkYTg5MS05ZGQwLTRjNzUtODJlMC02YWUyNGQwNGJhOGYucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI2MDExMSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNjAxMTFUMDk0NzI4WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZmJlM2I3ZmM0OGEwNTAyNDljZWU2ZjI5OTg1NGZhY2QwZTBlZTVmMjU5Y2QxODEyMGQ2NWU5YzE3YzU4OWZmZSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.gsNcDRKFCFK0unMmQHsAm8WJFRqAOUqaYU3-g-wHDag)
> 
> Hi there 👋,
> 
> I created a simple game where you have to guess the colors based on the name: [ColorGuesser.com](https://colorguesser.com/)
> 
> So far, I have a working prototype. Many things could be improved or added (I have written about this [here](https://kiruio.substack.com/i/126414175/incubating-colorguessercom)).
> 
> I am open for any feedback.
> 
> Thank you very much, Kiru

I tried it after a few attempts i was able to get this guess of a 99 accuracy
<img width="470" height="594" alt="image" src="https://github.com/user-attachments/assets/29ca984f-f656-40f9-b825-f7bd77008901" />
and that was in my best score i got being
<img width="527" height="431" alt="image" src="https://github.com/user-attachments/assets/779061ff-5f67-4494-9307-27bf561a17c4" />
462/500 but i could maybe get perfect in a few hundred more rounds and memory LOL
